### ISSUE TITLE
Suggestions on how to improve CSS

### ISSUE THREAD

**Issue by combe15** [devanshuyadav/helpage#44]
While looking at the project, I've noticed some quality of life issues with the CSS file or just how themes are implemented.

Consider using CSS at-rule: `@media: prefers-color-scheme` media feature.

This feature will automatically adjust theme settings without using any JS and it will always match the end user's selected preference. This will also prevent the Flash of White issue and the no theme issue when first asked for a name. You can learn more about this here: [zwbetz.com/fix-the-white-flash-on-page-load](https://zwbetz.com/fix-the-white-flash-on-page-load-when-using-a-dark-theme-on-a-static-site/)

If you insist on using the buttons for theme selection, then it is best practice to not use the class switcher with the ".light". Instead, just only use ".dark" when using dark mode while the light mode remains the default.

in short, if you insist on using buttons for theme:
| HTML code  | Theme |
| ------------- | ------------- |
| \<body class="light_dark"\>  | Light  |
| \<body class="light_dark dark"\>  | Dark  |
| \<body class="light_dark light"\>  | REMOVE  |

Also, the project's SCSS file is bloated. The file feels like someone just copied a regular CSS file and just renamed it into a .scss file. There are a lot of scss features not being taken advantage of.

**Comment by shiyasmohd** [comment]
First of all, I really liked your project. Its simple and cool. But i saw an issue in responsiveness of the site in mobile. So if you could create an issue (with hacktoberfest tag) and assign me , I would love to contribute.

**Comment by combe15** [comment]
@Shiyasmohd judging from the previous pull requests, don't ask for permission. Just make a PR with the changes you think are important.

But I'm not the repo owner. 🤷‍♂️

**Comment by devanshuyadav** [comment]
@combe15 that was the case earlier. But now since Unplanned PRs have started causing issues like this one. I'll only merge pre-assigned PRs.

**Comment by devanshuyadav** [comment]
> @combe15 that was the case earlier. But now since Unplanned PRs have started causing issues like this one. I'll only merge pre-assigned PRs. 


And Yes! One PR had the changes where CSS was just renamed to SCSS. I didn't think ahead and merged for the time being.
Also, I think removing the '.light' class won't be a problem. So If you'd like to work on it & also improve the SCSS, I'd assign it to you. Just let me know.

**Comment by devanshuyadav** [comment]
Out of Context:

> I'm not sure whether these PRs are considered for Hacktoberfest since I'm just a participant in Hactober. So if it is being updated just attach a screenshot for the same. It may get rejected after the 14 day maturity period. I can't assure that.
